rbi/annotations/activesupport.rbi

```diff
@@ -472,4 +472,15 @@ class ActiveSupport::ErrorReporter
     ).void
   end
   def report(error, handled: true, severity: T.unsafe(nil), context: T.unsafe(nil), source: T.unsafe(nil)); end
+
+  # @version >= 7.2.0.beta1
```

Comment:
I think you win the award for the very first version annotation? 🥇

Reply:
How do we want to handle runtime checks? I think CI will use the latest version however beta versions can only be specified in the Gemfile?

There's also the case of annotations with an incompatible version, `< 7.2.0beta1`. Should we just use `@method_missing` instead of dealing with this?

Reply:
I think this PR is fine thanks to https://github.com/Shopify/tapioca/pull/1585 but I'll leave it to @egiurleo before merging. We may wanna test versioning with apps that use lower Rails versions as we merge this.

Reply:
Yeah let's merge this and I'll add a note to our meeting on Monday to talk about it.